lib/index.js

```diff
@@ -18,7 +18,7 @@ internals.definition = {
         require: true
     },
     p: {
-        description: 'node_nodules path'
+        description: '`node_modules` path'
```

Comment:
Can you get rid of the backticks?  Otherwise looks good and thanks for letting me know!